teradata/CHANGELOG.md

```diff
@@ -1,2 +1,5 @@
 # CHANGELOG - Teradata
 
+## 1.0.0 / 2022-05-13
```

Comment:
No changelog entry?

Reply:
I think 0.0.1 was released unofficially, and no changes were added since then. I can manually add one

Reply:
Updated!

Reply:
Thanks! I wonder if this manual entry will be overridden?

Reply:
I think we've been able to overwrite changelogs in the past